@AH-64D Apache Longbow/Addons/fza_ah64_controls/config/cfgVehicles.hpp

```diff
@@ -1012,6 +1006,7 @@ class CfgVehicles
 		mainRotorSpeed = 1;
 		threat[] = {1,1,0.700000};
 		armor=60;
+		damageResistance=0.0055499999;
```

Comment:
How was this calculated?

Reply:
values are derived from vanilla blackfoot